Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16
+FROM node:17
```

Comment:
Why use a version that won't become LTS? Might as well go with 18, which *will* become LTS in like 2 weeks

Reply:
it breaks on 18

Reply:
Can always just keep it on 16 😄